inspirehep/modules/migrator/tasks.py

```diff
@@ -198,6 +198,7 @@ def add_citation_counts():
     from elasticsearch.helpers import bulk as es_bulk
     from elasticsearch.helpers import scan as es_scan
     from collections import Counter
+    from sets import Set
```

Comment:
In modern Python you can use the builtin `set` type, which is faster than the pythonic `sets.Set`